infrastructure/sandbox/JITProvisioner/lambda/go.mod

```diff
@@ -190,10 +190,6 @@ require (
 	howett.net/plist v1.0.0 // indirect
 )
 
-replace github.com/kolide/kit => github.com/zwass/kit v0.0.0-20210625184505-ec5b5c5cce9c
-
-replace github.com/micromdm/nanomdm => github.com/fleetdm/nanomdm v0.3.1-0.20230117193224-130d63f1d4ac
```

Comment:
Following discussion here https://github.com/fleetdm/fleet/pull/15861#issuecomment-1885267821 and Robert's reply, I went ahead and updated the `go.mod` of the sandbox infrastructure as well.